cmd/keycloakb/keycloak_bridge.go

```diff
@@ -836,6 +865,58 @@ func config(logger log.Logger) *viper.Viper {
 	return v
 }
 
+func checkMigrationVersion(logger log.Logger, dbParams dbConfig, minMigrationVersion string) error {
+
+	migrationVersionDBConn, err := dbParams.openDatabase()
+	defer migrationVersionDBConn.Close()
```

Comment:
defer after the *if* statement... you may have an invalid migrationVersionDBConn here (perhaps a nil value)

Reply:
Thanks